recipes/pymultinest/meta.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: pymultinest
+  version: "2.6"
+
+source:
+  fn: pymultinest-2.6.tar.gz
+  url: https://pypi.python.org/packages/c2/fc/77bdd8356068125bba0a5c07bfa2b4a3f492bff614aeae783bc1a566d699/pymultinest-2.6.tar.gz
+  md5: d9e5eee6abd3784056ca5ccc1f05cab9
+
+build:
+   number: 1
+   skip: true  # [win]
+   skip: true  # [py3k]
+
+requirements:
+
+  build:
```

Comment:
Are all of these packages required to `build` the package, or are they mostly just `run` requirements? I'm not sure how `setuptools` handles the `requires` argument:

https://github.com/JohannesBuchner/PyMultiNest/blob/master/setup.py#L25

as compared to `setup_requires` or `install_requires`

Also, I think you're going to need `Cuba`

Reply:
They need to be there for building, but I cleared up the selection (I missed some in the previous one, and since this is not a C++ extension I didn't need the `toolchain`)